pubsub/api/Pubsub.Samples.Tests/PubsubFixture.cs

```diff
@@ -115,6 +115,14 @@ public Subscription CreateSubscription(string topicId, string subscriptionId)
         return subscription;
     }
 
+    public Subscription CreateExactlyOnceDeliverySubscription(string topicId, string subscriptionId)
+    {
+        var createSubscriptionWithExactlyOnceDeliverySample = new CreateSubscriptionWithExactlyOnceDeliverySample();
+        var subscription = createSubscriptionWithExactlyOnceDeliverySample.CreateSubscriptionWithExactlyOnceDelivery(ProjectId, topicId, subscriptionId);
+        TempSubscriptionIds.Add(subscriptionId);
```

Comment:
Add the subscription ID to the temp list before creating the subscription. If creation fails client side, it might have still sucedded server side and we'd want to delete the subscription.

Reply:
Done.